test/plots/piePlotTests.ts

```diff
@@ -264,6 +264,16 @@ describe("Plots", () => {
         });
         svg.remove();
       });
+
+      it("formatters are used properly", () => {
+        piePlot.labelsEnabled(true);
+        piePlot.labelFormatter((n: number) => n + "%");
+        var texts = svg.selectAll("text")[0].map((n: any) => d3.select(n).text());
+        assert.lengthOf(texts, 2, "both texts drawn");
+        assert.strictEqual(texts[0], "5%", "first label is 5%");
+        assert.strictEqual(texts[1], "15%", "first label is 15%");
```

Comment:
Wrong message here :/

Also a more specific message might be "The formatter was used to format the <nth> label"